src/lib/components/Timeline/Timeline.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { Meta, Story } from '@storybook/react/types-6-0';
+import { HiArrowNarrowRight } from 'react-icons/hi';
+import { Timeline, TimelineProps } from '.';
+import { Button } from '../Button';
+
+export default {
+  title: 'Components/Timeline',
+  component: Timeline,
+} as Meta;
+
+const Template: Story<TimelineProps> = (args) => <Timeline {...args} />;
+
+export const DefaultTimeline = Template.bind({});
+DefaultTimeline.storyName = 'Default Timeline';
+DefaultTimeline.args = {
+  children: (
+    <Timeline>
```

Comment:
The children here should be only the array of timeline items since this will be passed to the template in line 11.